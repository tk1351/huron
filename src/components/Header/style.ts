import { css } from '@emotion/react'
import { GREEN } from '../../constants/constants'

export const header = css({
  backgroundColor: GREEN,
  width: '100%',
  height: '50px',
  padding: '0 170px'
})

export const link = css({
  textDecoration: 'none'
})
