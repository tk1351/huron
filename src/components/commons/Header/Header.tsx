import { FC } from 'react'
import { header, link } from './style'
import { APP_NAME } from '../../../constants/constants'

export const Header: FC = () => (
  <header css={header}>
    <h1>
      <a href="#" css={link}>
        {APP_NAME}
      </a>
    </h1>
  </header>
)
