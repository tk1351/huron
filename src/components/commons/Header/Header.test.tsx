import * as renderer from 'react-test-renderer'
import { Header } from './Header'

describe('Testing Header Component', () => {
  describe('Render', () => {
    it('正しくHeaderコンポーネントが描画されていること', () => {
      const App = renderer.create(<Header />).toJSON()
      expect(App).toMatchSnapshot()
    })
  })
})
