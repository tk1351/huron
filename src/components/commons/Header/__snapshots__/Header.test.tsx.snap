// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header Component Render 正しくHeaderコンポーネントが描画されていること 1`] = `
<header
  css={
    Object {
      "map": undefined,
      "name": "14hz84z",
      "next": undefined,
      "styles": "background-color:#066163;width:100%;height:50px;padding:0 170px;",
      "toString": [Function],
    }
  }
>
  <h1>
    <a
      css={
        Object {
          "map": undefined,
          "name": "ytumd6",
          "next": undefined,
          "styles": "text-decoration:none;",
          "toString": [Function],
        }
      }
      href="#"
    >
      Blog
    </a>
  </h1>
</header>
`;
